mysql-8.0/Dockerfile

```diff
@@ -0,0 +1,73 @@
+FROM    gitpod/workspace-full
+
+
+##
+## -- Set up the environment for a Docker build
+##    -----------------------------------------
+USER    root
+RUN     apt-get update 
+RUN     apt-get install debconf-utils -y -q
+ENV     DEBIAN_FRONTEND=noninteractive
+
+
+##
+## -- Install the Oracle Repo for MySQL
+##    ---------------------------------
+USER    root

```

Comment:
Note: No need to re-specify `USER` every time. Basically, in a Docker context, you have the same `USER` for all the instructions. You can change it mid-way e.g. with `USER gitpod`, and then it stays `gitpod` for all the rest of the instructions below.